components/button.json

```diff
@@ -21,6 +21,25 @@
         "text": "@theme-text-primary-disabled"
       }
     },
+    "primaryDark": {
```

Comment:
can we delete this group of tokens added then?

Reply:
Sorry about that, it was a leftover that I overlooked. Pushed commit [Removed primaryDark from button](https://github.com/momentum-design/tokens/pull/212/commits/9065f518a71c1a87f7bba90347ff00bb6e621d4e).
Changing this conversation to resolved.